pkg/telemetryquerylanguage/tql/boolean_value_test.go

```diff
@@ -15,116 +15,103 @@
 package tql
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql/tqltest"
 )
 
-func Test_newComparisonEvaluator(t *testing.T) {
```

Comment:
Note that I haven't deleted any tests, I've just condensed them into something easier to write and extend (see below).